src/avatar.js

```diff
@@ -0,0 +1,103 @@
+console.log(`Hubs version: ${process.env.BUILD_VERSION || "?"}`);
+
+// We should technically be able to just include three here, but our dependancies are tangled such that not having aframe is a bit difficult
+import "aframe";
+import "./utils/logging";
+
+import ReactDOM from "react-dom";
+import React from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+
+import { disableiOSZoom } from "./utils/disable-ios-zoom";
+disableiOSZoom();
+
+import { App } from "./App";
+
+import AvatarPreview from "./react-components/avatar-preview";
+import { WithHoverSound } from "./react-components/wrap-with-audio";
```

Comment:
We've stopped using this and we intend to remove it altogether. Let's not propagate it.